vsintegration/src/FSharp.VS.FSI/fsiSessionToolWindow.fs

```diff
@@ -530,19 +530,31 @@ type internal FsiToolWindow() as this =
         try
             showNoActivate()
             let directiveC  = sprintf "# 1 \"stdin\""    (* stdin line number reset code *)                
-            let text = "\n" + text + "\n" + directiveC + "\n;;\n"
+            let text() = "\n" + text + "\n" + directiveC + "\n;;\n"
             executeTextNoHistory null text
         with _ -> ()
 
+    let hide () =
+        if sessions.SupportsInteractivePrompt then
+            """#interactiveprompt "hide" """
+        else
+            ""
+
+    let show () =
+        if sessions.SupportsInteractivePrompt then
+            """#interactiveprompt "show" """
+        else
+            ""
+
     let executeInteraction dbgBreak dir filename topLine (text:string) =
-        let interaction = $"""
-#interactiveprompt "hide"
+        let interaction() = $"""
+{    hide()}
```

Comment:
```suggestion
{hide()}
```

Reply:
i'm afraid there is a parser bug. it's necessary unfortunately.